src/base64/base_common.rs

```diff
@@ -84,8 +84,11 @@ fn handle_input<R: Read>(
         wrap_print(&data, encoded);
     } else {
         match data.decode() {
-            Ok(s) => print!("{}", String::from_utf8(s).unwrap()),
+            Ok(s) => {
+                let r = stdout().write_all(&s);
+                r.ok();
```

Comment:
This should check if there is an error and crash (like below) if so instead of pretending nothing went wrong.

Reply:
Hi @Arcterus 

Thanks for reviewing.
My understanding is that an error can only occur when using a windows console for non-utf8 messages. I have written a crash message accordingly.

This issue kind of defeats the purpose of my commit on windows, but works (for me) on linux.
Once the portability issue of stdout().write_all() has been fixed base64 should present uniform behavior.
I have not looked at why there is this restriction on windows.

Also, the CI fails and tells me `fatal: Could not parse object 'e25e82618684c1f5add60773e8c3784e9ad5eaf7'.` 
Do you know what I should do about it ?

Best regards,